**Practical Gene expression, epigenetics and disease: Uncovering functional consequences of a risk variant**

*Friday 23th of January 2026*

*General remarks:*
- We recommend you think about the question and try to get a (first idea of an) answer first. Then, if preferred, you can use the internet to confirm your answer. This approach helps you learn much better, and the answer will stay in your mind much longer.
- Use the GRCh37/hg19 assembly of the reference genome across this assignment.

*Save your answers in a presentation:*
- Store your answers in a PowerPoint (i.e., \*.pptx, or open source alternative) file 
- Make sure to save the final presentation under your full name and follow this pattern: date\_firstname\_lastname.pptx (e.g. 20260129_Jane_Doe.pptx). Date is in YYYYMMDD format.

*Details on how to answer questions:*
- The title of each slide should specify which question (**Q1**, **Q2** etc.) is being answered.
- If a coordinate is requested, an approximation is sufficient (+/- 1kb). You don't need to provide these coordinates at a base pair resolution.
- If a screenshot is requested, paste the picture into a slide in your presentation.
- If you are asked to take a screenshot of a webpage, but the webpage is larger than a single page, you can take the screenshot of the most relevant part of the page.

**Aim**: To experimentally validate the functional relevance of a candidate genetic risk variant and identify the likely affected gene.

**Background**: Let us assume that rs9749546, a SNP (single nucleotide polymorphism, has been associated with sickle cell anemia in a GWAS (genome wide association study) in Nigerian individuals.

**Q1.** What is the general aim of a GWAS? Describe the possible study design of this sickle cell anemia GWAS. How would this study design differ if we wanted to identify variants that are associated with height, instead of risk of hemophilia? Give your answer in a few sentences or bullet points. You can add pictures to illustrate your answer (sketches you made yourself, or figures from the internet)

**Q2.** What is the genomic coordinate of rs9749546? In which gene is this variant located?

*Hints:*
- You can use the [UCSC Genome Browser](http://genome-euro.ucsc.edu/) for this question
- At the top left of the screen, hover over “Genomes” and click the correct genome
- We advise to clear your environment on the website and start from scratch ('hide all')
- Add a “gene track” so that we can see genes and their genomic locations on the browser. Search for “GENCODE V47lift37” on the web page. You will see a drop-down list underneath. Click on it and choose “pack”. Then, on the right side of the page click on “refresh”.
- Next, we would like to find our variant of interest. On the top center of the page, there is a search box which says: “gene, chromosome range, …”
- You will see many links pointing to the same variant in different databases or a similar variant in different species
- Besides each link, you can also see the corresponding coordinate of that variant
- We will work with the variant shown below “Common (1000 Genomes Phase 3 MAF >= 1%) Short Genetic Variants from dbSNP Release 153”
- Note its genomic coordinates in your presentation and click on the corresponding link
- Once the genome browser loads again, you will see a tiny highlighted vertical line that indicates the location of our variant
- On the top of the page, you can see “Zoom in” and “Zoom out” buttons. Adjust the zoom (by zooming out) to see multiple genes surrounding our variant.
- Take a screenshot such that the surrounding (4 to 8) genes and the variant of interest are seen
- To reduce the clutter in the web page, you can hide the “variant track”. To do this, find the gray vertical bar on the left side of the track, right click on it, and choose “hide track set”.
- Note that the highlighted vertical bar is still present

As seen in **Q2**, rs9749546 is in a non-coding part of the genome. As no gene is directly hit with this variant, the functional impact of this variant is therefore not immediately clear to us. We hypothesize that this region may harbor an enhancer that influences the expression of its surrounding genes. 

**Q3.** Which histone acetylation mark is most commonly used as an indicator of an active enhancer? As follow-up, name one or two other histone marks that are also often associated with active enhancers.

**Q4.** Q4. Enhancer activity is cell-type specific. For this exercise, we will use K562, a human erythroleukemia cell line that models blood/erythroid (red blood cell) biology, as a relevant system for hemoglobin-related disease. Using K562 epigenomic data, does rs9749546 overlap an active enhancer in K562?

*Hints:*
- Load the relevant ChIP-seq track for this histone mark and inspect the signal/coverage at the variant. For this exercise, we will treat any clear coverage at the variant position as evidence for an enhancer.
- In the UCSC Genome Browser, click Track search.
- In the search box, type: K562 <histone modification name>.
- Select the track described as “H3K27Ac Mark (Often Found Near Regulatory Elements) on K562 Cells from ENCODE”.
- Set Visibility to Full (hide the other tracks), then click View in Browser.
- Interpret the track at a broad level: focus on the overall signal around the locus rather than individual small peaks; the number of peaks is not important here.
- Adjust the zoom so you can see both the variant position and the ChIP-seq coverage at/around it.
- Take a screenshot showing the variant’s chromosomal position and the ChIP-seq signal at that location.

**Linkage disequilibrium** 

At certain positions in the genome, we commonly see differences between individuals. SNPs are an example of such a common variant: some people have the reference allele (this is the same allele as in the reference genome), others have the alternative allele (NB: SNPs are always defined by their location and their alternative allele, so if there is three possible alleles at one position there would be 2 possible SNPs at this position). SNPs are not fully independent of each other: genomic variations tend to be inherited together in groups. This is due to recombination: the process during meiosis that ensures there is one (recombined) copy of each chromosome in egg and sperm cells. Genomic recombination is not random: the genome tends to break more often in certain places. SNPs that are in between places where the genome often recombines are therefore inherited together more often. If two SNPs are often inherited together, we say they are in linkage disequilibrium (LD). The recombination rate across the genome differs between populations. Therefore the LD between the same SNPs is different in different populations. Because SNPs are not inherited independently, we do not need to sequence every base of the genome to identify all variants. Even if we sequence some of these variants (which reduces cost of sequencing), we can make the full overview as long as we know which variants are likely to be inherited together. In our case, even if no gene or regulatory element is hit, other variants that are in LD with our variant can have a functional impact. Consequently, it is important to know which variants are in LD with our variant of interest.

**Q5.** Knowing about linkage disequilibrium, how can we reduce the sequencing cost in a GWAS study? 

Hints:
- To answer this, you can look up how GWAS is done and what exactly is being measured.

**Q6.** Give an approximation of the start and end coordinate of the 'LD group' that encloses our SNP.

Hints:
- Go to: https://ldlink.nih.gov/ldproxy, fill in the SNP id in the 'Variant' field, and the relevant population in the 'Population' field. Under 'Annotation', click 'None' and click on 'Calculate'. Look at the plot that shows up.
- Which population do you need?
- You can use a threshold of an R^2 higher than 0.4 to identify all SNPs that are in LD with the variant of interest.
- If you hover over a SNP in the plot, you see the distance from the index SNP.

**Q7.** Identify the coordinates of the potentially affected enhancer(s). Take a screenshot of the ChIP-seq track and annotate the regions in powerpoint that you think are related to the enhancer.

Hints:
- You can try to load the relevant ENCODE ChIP-Seq in the UCSC genome browser (as you did before in **Q4**)
- Use the relevant LD-block coordinate (found in **Q6**) to find the neighbor enhancers in the same LD-block. To add the LD-block coordinate as track, " Manage custom tracks" > "Add custom tracks" >Add the coordinates in the data box, e.g. chr1 start end (use this format, so make sure to add chr and spaces between chr, start and end)> Submit.
- Note that it is fine if you combine multiple enhancers and consider them as one. The exact number of identified enhancers is not the point here.

\-----------------------------------

Now we know which enhancers are most likely affected by this variant. The next step is to determine the most likely affected genes. For this, we want to understand which genes co-localize with the affected enhancers within the 3D space of the nucleus. In other words, they are residing together in a Topologically Associating Domain (TAD). 

**Q8.** What are the chromosomal coordinates of the affected TAD(s)? 

Hints:

- Go back to UCSC genome browser. 
- Use “track search” button and search for: “K562 Hi-C”
- Check the box beside “K562 Hi-C”, make sure the “Visibility” is set to “full”
- Click on “View in Browser”.
- Make sure you are limiting the UCSC genome browser to coordinates of the LD-block (you got this coordinate in **Q6**).
- In the “zoom out” section of UCSC Genome Browser on the top right, click on the Zoom out “10x” button. Your coordinates should be roughly chr19:18,236,797-18,836,506.
- Configure the track (by right-clicking on the recently loaded Hi-C track) and selecting “Configure K562 Hi-C”. Specifically, in the opened window:
  - Select “KR” in the “Score normalization” section. 
  - For “Draw mode use triangle”.
  - Use Maximum: 50.
  - For “Resolution” select “5.0 kB”.
  - Click OK.
- Make a screenshot of the TAD(s).

**Q9.** Consider varying the other parameters that could be changed in the HiC track’s configuration panel (these configurations can be accessed by right-clicking on the HiC track). Describe on a slide, the aim of at least one configuration and how it changes the shown track.

**Q10.** Using the identified TAD(s), determine the most likely set of genes that are affected by our variant of interest. Mention their name in the corresponding slide.

\--------------------------------------

We can perform additional confirmatory analyses to verify that the identified TADs are correct. One example analysis could be to check and see if the TADs borders are bound by genomic elements that restrict chromatin interactions (and form TADs as a result). In particular, we want to validate the presence of bound architectural proteins at the TAD borders.


**Q11.** Which architectural protein is typically found to be bound at TAD boundaries? Is this factor present at our TAD(s) boundaries? Take a screenshot of the TAD in a way that the loaded factor track is also visible.

Hints:

- Using “track search” functionality, search for: “K562 *<Factor Name>*”.
- Many tracks will be shown, choose the one that has the following description “ChIP-seq Signal from ENCODE/Broad”. With this, we are loading a track with maximum details (hence the name “Signal”), instead of only looking at the peaks in the data.
- Make sure the “Visibility” is set to full.
- Click on “View in browser” or “Return to Browser”.

**------------------------------------**

For a gene to be controlled by enhancers within a TAD, it should be an actively expressed gene. If a gene is inactive (i.e., silent), it is unlikely to be affected by a non-functional enhancer (or a variant). We will look at the expression of the genes located within our region of interest in the genome in blood.

**Q12.** Which genes are active (or expressed) in blood? Take a screenshot of the recently loaded track in which genes and their expression in the tissue of origin is shown. No need to include all loaded tracks in this screenshot.

Hints:	

- Use the GTEx expression track for this. In the UCSC Genome Browser, scroll down until you find the “Expression” section. Look for “GTEx Gene V8” track. Once found, click on the text to go to its configuration page.
- Underneath, there are many check boxes. Click on “clear all” button to make sure all check boxes are inactive.
- Click on “Whole\_Blood” mark and make it active.
- On the top of the page, look for a dark blue “Go” button at the center. Before clicking the button, make sure the “full” option on its left is selected.
- In this visualization, each bar represents expression of a gene in “Whole Blood” tissue.

\--------------------------------------

Further, we hypothesize that the affected gene is unlikely to be a housekeeping gene (i.e., a gene that is ubiquitously expressed across tissues) and it should be a more tissue-restricted gene that is causing the disease (e.g., leukemia). To examine this, we will compare expression of the candidate genes across multiple cell types.

Hints:

- You can again use “GTEx Gene V8” track.
- Again, click on “GTEx Gene V8” link that is located below the visualized tracks to start the configuration.
- This time, select the following tissue types:
  - Brain-Cortex
  - Liver
  - Pancreas
  - Heart - Atrial Appendage
  - Skin - Not Sun Exposed (Suprapubic)
  - Whole Blood
- In this visualization, each bar represents expression of a gene in a particular cell-type. Each color represents gene expressions of a specific cell-type. Using this visualization, you can easily see the expression levels of genes in multiple cell-types at the same time.


**Q13.** What are the remaining candidate genes that are expressed in a more tissue-restricted manner (i.e., mostly active *only* in the whole blood and no other tissue types)?


**Q14**

you have identified a potential candidate enhancer and gene. How would you verify whether the enhancer is indeed regulating the potential target gene in K562 cells? How would you identify disease-causing variants in the enhancer?




